core/src/main/scala/cats/Applicative.scala

```diff
@@ -37,6 +37,10 @@ import simulacrum.typeclass
   def sequence[G[_], A](as: G[F[A]])(implicit G: Traverse[G]): F[G[A]] =
     G.sequence(as)(this)
 
+  /** one or none */
```

Comment:
I think we need more comments than this. For `F = List` this will be `one and none` (we will always concatenate None to the list), so it behavior seems highly dependant on the `F`.